gwvolman/lib/dataone/metadata.py

```diff
@@ -200,7 +200,7 @@ def add_object_record(self, root, name, description, size, object_format):
         self.create_format(object_format, physical_section)
         ET.SubElement(entity_section, 'entityType').text = 'dataTable'
 
-    def set_user_name(self, root, first_name, last_name):
+    def set_user_name(self, root, first_name, last_name, contact_info=None):
```

Comment:
Just for understandability, I think `contact_info` should be `userId`, since that's what's being set in the EML